sormas-ui/src/main/java/de/symeda/sormas/ui/labmessage/LabMessageEditForm.java

```diff
@@ -52,7 +52,7 @@ protected void addFields() {
 		addFields(LabMessageDto.UUID, LabMessageDto.MESSAGE_DATE_TIME);
 		labMessageDetails = new Label();
 		Panel detailsPanel = new Panel(labMessageDetails);
-		detailsPanel.setWidth(550, Unit.PIXELS);
+		detailsPanel.setHeight(500, Unit.PIXELS);
```

Comment:
I would propose to leave the fixed width.
How did you get to setting the height to 500 pixels?  I would prefer if the heigt was set to 100% (which will relate to available screen height)...

Reply:
If I set the height to 100% then the panel displays the entire messages, without a vertical scroll bar, and the scroll bar of the layout will be used. In this case, we have to scroll to the bottom to see the horizontal scroll bar of the panel.

Reply:
Does every element in the hierarchy above the panel have the height set to 100%. That is necessary for the panel's 100% to work...